app/javascript/Components/ta_table.jsx

```diff
@@ -59,87 +113,7 @@ class TATable extends React.Component {
   render() {
     const {data} = this.state;
     return (
-      <ReactTable
-        data={data}
-        columns={[
-          {
-            Header: I18n.t("activerecord.attributes.user.user_name"),
-            accessor: "user_name",
-          },
-          {
-            Header: I18n.t("activerecord.attributes.user.first_name"),
-            accessor: "first_name",
-          },
-          {
-            Header: I18n.t("activerecord.attributes.user.last_name"),
-            accessor: "last_name",
-          },
-          {
-            Header: I18n.t("activerecord.attributes.user.email"),
-            accessor: "email",
-          },
-          {
-            Header: I18n.t("roles.active") + "?",
-            accessor: "hidden",
-            Cell: ({value}) => (value ? I18n.t("roles.inactive") : I18n.t("roles.active")),
-            filterMethod: (filter, row) => {
-              if (filter.value === "all") {
-                return true;
-              } else {
-                return (
-                  (filter.value === "active" && !row[filter.id]) ||
-                  (filter.value === "inactive" && row[filter.id])
-                );
-              }
-            },
-            Filter: selectFilter,
-            filterOptions: [
-              {
-                value: "active",
-                text: `${I18n.t("roles.active")} (${this.state.counts.active})`,
```

Comment:
@chickenwaddle77 one other thing I forgot to mention: because this part is being deleted, you can also get rid of the `counts` state attribute, which is no longer used.